libpod/container_internal_linux.go

```diff
@@ -1494,7 +1494,7 @@ func (c *Container) generateResolvConf() (string, error) {
 	destPath := filepath.Join(c.state.RunDir, "resolv.conf")
 
 	if err := os.Remove(destPath); err != nil && !os.IsNotExist(err) {
-		return "", errors.Wrapf(err, "error removing resolv.conf for container %s", c.ID())
+		return "", errors.Wrapf(err, "container %s", c.ID())
```

Comment:
Again, this one should probably stay

Reply:
The error will look like 
container ABCDEF: failed to remove PATHTO/etc/resolv.conf permission denied.
Rather then 
error removing resolv.conf for container ABCDEF:  failed to remove PATHTO/etc/resolv.conf permission denied.